draft-webtransport-http2.md

```diff
@@ -5,7 +5,7 @@ docname: draft-webtransport-http2-latest
 
```

Comment:
Line 415: "SHOULD refresh..." - Or perhaps, "SHOULD take measures to prevent a peer or intermediary from timing out the Connect Stream" ?

Reply:
Nice! Added